### PR TITLE
today as datetime function

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @MichaelButkovic

### CODE REVIEW

**Comment by benghaziboy** [comment]
Ran into some issues with the configurable `TODAY`

File: bulbs/utils/methods.py [hunk 1]
```diff
@@ -18,6 +19,14 @@ def today():
     return getattr(settings, "TODAY", get_central_now())
 
 
+def today_as_datetime():
+    """Datetime/Date comparisons aren't great, and someone might configure TODAY, to be a date."""
+    now = today()
+    if not isinstance(now, datetime) and isinstance(now, date):
+        now = datetime.combine(now, datetime.min.time())
+        now = now.replace(tzinfo=tz.gettz('America/Chicago'))
+
```
Comment (mparent61) [inline bulbs/utils/methods.py]: return now???

Comment (benghaziboy) [inline bulbs/utils/methods.py]: lol. I already caught this. whoops

Comment (mparent61) [inline bulbs/utils/methods.py]: classic Lowe boy test coverage

**Pull request merged**